Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/LightingOptions.azsli

```diff
@@ -8,15 +8,65 @@
 
 #pragma once
 
+// --- Light Defines ---
```

Comment:
It would be nice to have a comment here that explains why we have both ENABLE_ preprocessor flags as well as shader options for controlling these features.

Reply:
Will add